Soccar [Game-Server]/src/nl/soccar/gameserver/util/CarUtilities.java

```diff
@@ -12,13 +12,22 @@
 import java.util.List;
 
 /**
+ * Utilities for the Car.
+ *
  * @author PTS34A
  */
 public final class CarUtilities {
 
     private CarUtilities() {
     }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)